src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -502,76 +486,157 @@ export const BorrowInput = ({
     lendingQuoteError?.message,
   ])
 
+  const handleBorrowSubmit = useCallback(() => {
+    return !!isUnsafeQuote ? setShouldShowWarningAcknowledgement(true) : onSubmit()
+  }, [isUnsafeQuote, onSubmit])
+
   if (!(collateralAsset && borrowAsset && collateralFeeAsset)) return null
 
   return (
     <SlideTransition>
-      <Stack spacing={0}>
-        <TradeAssetInput
-          accountId={collateralAccountId}
-          assetId={collateralAssetId}
-          assetSymbol={collateralAsset.symbol}
-          assetIcon={collateralAsset.icon}
-          onChange={handleDepositInputChange}
-          onToggleIsFiat={toggleCollateralAssetIsFiat}
-          isFiat={collateralAssetIsFiat}
-          cryptoAmount={depositAmountCryptoPrecision ?? '0'}
-          fiatAmount={fiatDepositAmount ?? '0'}
-          isAccountSelectionDisabled={isAccountSelectionDisabled}
-          isSendMaxDisabled={false}
-          percentOptions={percentOptions}
-          showInputSkeleton={false}
-          showFiatSkeleton={false}
-          label={translate('lending.depositAsset', { asset: collateralAsset.symbol })}
-          onAccountIdChange={handleCollateralAccountIdChange}
-          formControlProps={formControlProps}
-          layout='inline'
-          labelPostFix={collateralAssetSelectComponent}
-        />
-        <Flex alignItems='center' justifyContent='center' my={-2}>
-          <Divider />
-          <IconButton
-            isRound
-            size='sm'
-            position='relative'
-            variant='outline'
-            borderColor='border.base'
-            zIndex={1}
-            aria-label={translate('lending.switchAssets')}
-            icon={swapIcon}
+      <WarningAcknowledgement
+        message={`This borrow has high slippage (${bnOrZero(quoteSlippageDecimalPercentage)
+          .times(100)
+          .toFixed(2)
+          .toString()}%). Proceed with caution.`}
```

Comment:
same here, translation?

Reply:
It is, though I've moved it in https://github.com/shapeshift/web/pull/6647/commits/05c292f7035995dab341ddf89c2ad7e64e3e383b for consistent grouping.